doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -432,62 +410,50 @@ Instructions
 
    .. note::
       All customized scenario XMLs will be in user-defined groups, which are
-      located in ``misc/vm_configs/xmls/config-xmls/[board]/user_defined/``.
+      located in ``misc/config_tools/data/[board]/user_defined/``.
```

Comment:
I think this is something we should change. We shouldn't add files to the source code that's under the git version control. We should always ask the user where to put these. If we want to use a default value, let's use the user's home directory.

We do not need to fix this in this PR obviously :-)

Reply:
Agree. As we have previously mentioned, in v2.5 we plan to tweak the configuration editor a bit so that it can be invoked by Makefile to edit the configuration in the build directory, just like how ``menuconfig`` works in Linux.